ui/app/src/utils/i18n-legacy.ts

```diff
@@ -142,6 +142,13 @@ const contentTypesMessages = defineMessages({
 
 });
 
+const commonMessages = defineMessages({
+  notification: {
+    id: 'commonMessages.notification',
+    defaultMessage: 'Notification'
+  }
+});
+
```

Comment:
Let's rename this to words. Only single words may go in here.

```js
const words = defineMessages(
  ...
  id: 'words.notification'
  ...
)
```

Carry this over to the history PR (and others as required).

Reply:
done, history PR will be updated as well